Example/__tests__/SharedValue.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { View, Button } from 'react-native';
+import { render, fireEvent } from '@testing-library/react-native';
+import { useSharedValue } from 'react-native-reanimated';
+
+const TestComponent1 = () => {
+  const sv = useSharedValue(1);
+  return <View testID="view">{sv.value}</View>;
+};
+
+function TestComponent2() {
+  const sv = useSharedValue(1);
+
+  return (
+    <View>
+      <View testID="view">{sv.value}</View>
+      <Button
+        testID="button"
+        title="run"
+        onPress={() => {
+          sv.value = sv.value + 1;
+        }}
+      />
+    </View>
+  );
+}
```

Comment:
I think naming them `ViewTestComponent` and `TestComponentWithSV` (or something along those lines) would be better.